certification/pyxis/pyxis.go

```diff
@@ -299,7 +299,19 @@ func (p *pyxisClient) GetProject(ctx context.Context) (*CertProject, error) {
 }
 
 func (p *pyxisClient) updateProject(ctx context.Context, certProject *CertProject) (*CertProject, error) {
-	b, err := json.Marshal(certProject)
+	// We cannot send the project type to pyxis in a Patch.
+	// Copy the CertProject and strip the Type value to
+	// have omitempty skip the key in the JSON patch.
+	patchCertProject := &CertProject{
```

Comment:
Just a suggestion, unless you want to be as explicit as you are. Feel free to ignore.

```suggestion
	patchCertProject := *certProject
	patchCertProject.Type = ""
```